custom_components/babybuddy/switch.py

```diff
@@ -0,0 +1,288 @@
+"""Platform for Baby Buddy binary switch integration."""
+from __future__ import annotations
+
+import logging
+from datetime import datetime, time
+from typing import Any
+
+import homeassistant.util.dt as dt_util
+import voluptuous as vol
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_ID, ATTR_NAME, CONF_HOST
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers import entity_platform
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import BabyBuddyCoordinator
+from .client import get_datetime_from_time
+from .const import (
+    ATTR_ACTIVE,
+    ATTR_AMOUNT,
+    ATTR_CHILD,
+    ATTR_END,
+    ATTR_FEEDINGS,
+    ATTR_FIRST_NAME,
+    ATTR_LAST_NAME,
+    ATTR_METHOD,
+    ATTR_MILESTONE,
+    ATTR_NOTES,
+    ATTR_SLEEP,
+    ATTR_START,
+    ATTR_TIMER,
+    ATTR_TIMERS,
+    ATTR_TUMMY_TIMES,
+    ATTR_TYPE,
+    DEFAULT_FEEDING_METHOD,
+    DEFAULT_FEEDING_TYPE,
```

Comment:
these can be removed now

Reply:
I am only using these now for the `select` entities. These help a user know the available options for some fields that require specific values (like feeding type and method). The default value is only loaded when the integration is first added. Afterwards the last selected option is restored upon restart.

Reply:
Gotcha. [I actually prefer this explanation.](https://www.home-assistant.io/blog/2021/07/07/release-20217/#new-entity-select)

I'm okay with this. But, please remove the init default values.